Yoast/wordpress-seo
# How-to and FAQ blocks: remove the bold button from the bolded fields

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

- removes the Bold button from the How-to step name and the FAQ question RichText "fields"
- also changes `onSetup` to `unstableOnSetup` as the former will be deprecated in Gutenberg 3.9.0
- solves a small CSS issues: when removing a placeholder and typing, the fields "jumped" by a few pixels because of different line-height values

Screenshots: 

When editing a "bolded" field, there's no Bold button in the toolbar:

<img width="736" alt="screen shot 2018-09-12 at 11 25 47" src="https://user-images.githubusercontent.com/1682452/45416393-27869280-b680-11e8-9e91-7308bc0ccc65.png">

When editing the other fields, the Bold button is available as usual:

<img width="696" alt="screen shot 2018-09-12 at 11 25 59" src="https://user-images.githubusercontent.com/1682452/45416428-3b31f900-b680-11e8-92e2-a5cee45cd2cb.png">

Note:
the default formatting buttons are `[ 'bold', 'italic', 'strikethrough', 'link' ]` so they can be configured for each RichText component. It is also possible to remove all of them as Gutenberg does in a couple places e.g. `formattingControls={ [] }`

## Test instructions
- first, **test on trunk**, build assets, add a How-to and a FAQ block 
- save the post
- switch to this branch, build assets, refresh, verify there are no "invalid block" warnings
- test all the functionalities of the How-to and FAQ blocks, they should work as before
- remove the blocks, re-add them, test again
- evaluate the UX, when editing the blocks bolded fields 


Fixes #10938
